src/lib/TokenService.js

```diff
@@ -0,0 +1,13 @@
+const setToken = (value) => {
+  localStorage.setItem('token', value);
+};
+
+const getToken = () => {
+  return localStorage.getItem('token');
+};
+
+const checkToken = () => {
+  return getToken() !== null && getToken().length !== 0;
+};
+
+export { setToken, getToken, checkToken };
```

Comment:
Awesome work on abstracting the token logic in a separate file. 👌🏻

Reply:
Yes I agree with Luka. This is really good way of approaching it. Clear and concise.

Reply:
Abstraction approaches / encapsulation will become super important when you deal with testing and more complex projects, this is awesome.